hapi-proto/src/main/proto/ResponseCode.proto

```diff
@@ -214,4 +214,5 @@ enum ResponseCodeEnum {
   SCHEDULE_WAS_DELETED = 203; // The Scheduled entity was already deleted
   INVALID_SCHEDULE_PAYER_ID = 204; // The provided Scheduled Payer does not exist
   INVALID_SCHEDULE_ACCOUNT_ID = 205; // The Schedule Create Transaction TransactionID account does not exist
+  SCHEDULE_MEMO_TOO_LONG = 206; // The Schedule Create Transaction Memo is longer than 100 bytes
```

Comment:
memo's for different entities (account, topic, token, schedule) will have the same errors. Lets rename this error to `ENTITY_MEMO_TOO_LONG` and edit the comment for it `// Entity memo size exceeded 100 bytes`